gatorgrade/output/src/output_functions.py

```diff
@@ -0,0 +1,57 @@
+"""
+This class is used for storing the main functions requested from the Github.
+
+Issue Tracker for the output team.

```

Comment:
Is this and the previous line of the docstring supposed to be the same sentence? Either way, can you fix the way this sentence is written?

Reply:
We updated our docstrings to be the correct format and more descriptive, thanks!